### PR TITLE
Feat minimap

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- adds Minimap
- adapts Map to move Minimap indicator, and to be moved by clicks on Minimap
- uses a movement strategy adapted from Map
- still has some window resize issues, responsiveness issues. should be addressed in another branch once the Map resize/responsive issues are fixed.

*Note: the Pan zone overlaps the Minimap. To test clicks you will need to reposition `#minimap` temporarily.

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -486,4 +489,68 @@ Site.Map = {
...
+
+    // Multiply map position by minimap scale.
+    // Map moves to negative coordinates,
+    // Minimap indicator moves positive
+    var minimapX = x * -(_this.minimapScale);
+    var minimapY = y * -(_this.minimapScale);
+
+    // If we don't know the indicator's current postion we get it
+    if (!_this.indicatorPosition) {
+      _this.indicatorPosition = Site.Map.getMapPosition(_this.minimapIndicator);
```
Comment (varffs) [inline src/js/main.js:567]: I think the use of `getMapPosition` to return the position of an element passed to it means the function should be renamed

**Comment by varffs** [comment]
Looks great and works except the minimap needs to be like the header children an area that pauses the pan of the map. Each time I click on something I get panned to the bottom left from it straight away.

**Review by varffs — CHANGES_REQUESTED** [review]
Looks great and works except the minimap needs to be like the header children an area that pauses the pan of the map. Each time I click on something I get panned to the bottom left from it straight away.

**Comment by mikeyrayvon** [comment]
i mentioned that in the description of the PR

File: src/js/main.js (hunk 1 above)
Comment (mikeyrayvon) [reply src/js/main.js:567]: ok

**Comment by varffs** [comment]
Sorry yes I see. I guess I feel that fix should be part of this. But it would involve a minor refactor of the ignore pan function so if you think that is another branch thats fine

**Pull request merged**